recipe/meta.yaml

```diff
@@ -22,18 +22,19 @@ requirements:
   run:
     - python >={{ python_min }}
     - typing_extensions >=3.7.4
+    # most restrictive
     - urllib3 >=1.26.5
-    - tqdm >=4.64.1
     - certifi >=2019.11.17
-    - grpcio >=1.59.0
     - pinecone-plugin-interface >=0.0.7,<0.0.8
-    - pinecone-plugin-inference >=2.0.0,<4.0.0
     - python-dateutil >=2.5.3
   run_constrained:
-    - googleapis-common-protos >=1.53.0
+    - googleapis-common-protos >=11.66.0
+    # most restrictive
+    - grpcio >=1.68.0
```

Comment:
This is listed upstream as optional:
https://github.com/pinecone-io/pinecone-python-client/blob/v6.0.2/pyproject.toml#L48
So I moved it here but maybe there was a reason to require it that I didn't see.